docs/content.zh/docs/concepts/overview.md

```diff
@@ -24,86 +24,93 @@ specific language governing permissions and limitations
 under the License.
 -->
 
-# Overview
-Flink Kubernetes Operator acts as a control plane to manage the complete deployment lifecycle of Apache Flink applications. Although Flink’s native Kubernetes integration already allows you to directly deploy Flink applications on a running Kubernetes(k8s) cluster, [custom resources](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) and the [operator pattern](https://kubernetes.io/docs/concepts/extend-kubernetes/operator/) have also become central to a Kubernetes native deployment experience.
+<a name="overview"></a>
 
-Flink Kubernetes Operator aims to capture the responsibilities of a human operator who is managing Flink deployments. Human operators have deep knowledge of how Flink deployments ought to behave, how to start clusters, how to deploy jobs, how to upgrade them and how to react if there are problems. The main goal of the operator is the automation of these activities, which cannot be achieved through the Flink native integration alone.
+# 概述
+Flink Kubernetes Operator 扮演控制平面的角色，用于管理 Apache Flink 应用程序的完整部署生命周期。尽管 Flink 的原生 Kubernetes 集成已经允许你直接在运行的 Kubernetes(k8s) 集群上部署 Flink 应用程序，但 [自定义资源](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) 和 [operator模式](https://kubernetes.io/docs/concepts/extend-kubernetes/operator/) 也已成为 Kubernetes 本地部署体验的核心。
 
-## Features
-### Core
+Flink Kubernetes Operator 通过自定义资源定义（CRD）来扩展 Kubernetes API，以便通过本地 k8s 工具（如 kubectl）管理和操作 Flink 部署。Operator 的核心功能包括：
+
+<a name="features"></a>
+
+## 特征
+
+<a name="core"></a>
+
+### 核心
 - Fully-automated [Job Lifecycle Management]({{< ref "docs/custom-resource/job-management" >}})
```

Comment:
just a minor sus: why not do translation for these lines ?

Reply:
At first, I thought it was good not to translate. I translated Fully-automated, but I didn’t think it was necessarily appropriate.